docs/src/further_topics/ugrid/data_model.rst

```diff
@@ -0,0 +1,319 @@
+.. include:: ../../common_links.inc
+
+.. _ugrid model:
+
+The Mesh Data Model
+*******************
+
+.. important::
+
+        This page is intended to summarise the essentials that Iris users need
+        to know about meshes. For exhaustive details on UGRID itself:
+        `visit the official UGRID conventions site`__.
+
+Evolution, not revolution
+=========================
+Mesh support has been designed wherever possible to fit within the existing
+Iris model. Meshes concern only the spatial location of data, and can
+optionally be limited to just the horizontal location (e.g. X and Y). Other
+dimensions such as time or experimental re-runs (and often vertical levels)
```

Comment:
```suggestion
dimensions such as time or ensemble member (and often vertical levels)
```

Reply:
I found the "suggest changes" button! Most will be super picky/pedantic - feel free to ignore.

Reply:
1d94cc892fcefdc2da2abdd32e593d1fb659d3b8